pkg/test/environment.go

```diff
@@ -1383,6 +1456,11 @@ func (e *Environment) NewGatewayConfig() *v1beta1.GatewayConfig {
 				GRPCListenAddress: fmt.Sprintf("tcp://localhost:%d", e.ports.ManagementGRPC),
 				HTTPListenAddress: fmt.Sprintf("localhost:%d", e.ports.ManagementHTTP),
 				WebListenAddress:  fmt.Sprintf("localhost:%d", e.ports.ManagementWeb),
+				// WebCerts: v1beta1.CertsSpec{
+				// 	CACertData:      dashboardCertData,
+				// 	ServingCertData: dashboardCertData,
+				// 	ServingKeyData:  dashboardKeyData,
+				// },
```

Comment:
keep or remove?

Reply:
Keep, I want to be reminded about this later on. lol